interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a program designed for version control. Github is a website that utilizes git and allows for collaborative programming on projects.
 
-Researched answer:
+Researched answer: Git is a free and open-sourced program that provides a version control system and can handle both small and large projects quickly and effectively. Github is a company that provides a cloud-based git repository hosting service. It was designed for the storing, tracking, and collaboration on software projects of all different sizes.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The Javascript operators that will return a Boolean value are >, <, <=, =>, ==, ===, ||, and &&. 
 
-Researched answer:
+Researched answer: Within Javascript, there are three operators that will return a boolean value. Those operators are AND, OR, and NOT. They can be used in a variety of ways and are very useful in the construction of complex logic components and flow.
```

Comment:
Wouldn't hurt to say what the names of those operators are (logical, relational, equality).